doc/userguide/partials/eve-log.yaml

```diff
@@ -280,3 +284,14 @@ outputs:
         #   event-set: false                # log packets that have a decoder/stream event
         #   state-update: false             # log packets triggering a TCP state update
         #   spurious-retransmission: false  # log spurious retransmission packets
+        #
+heartbeat:
```

Comment:
Should this not belong in outputs section ?

Reply:
The heartbeat *can be* a general mechanism for event scheduling. Currently, it only contains an output-related value.

It *could be* moved if the generality of the mechanism isn't needed.

Thoughts?

Reply:
How do FlowRecycler or FlowManager work ? Why is it different here ?